ludwig/utils/misc_utils.py

```diff
@@ -140,3 +144,39 @@ def set_saved_weights_in_checkpoint_flag(config):
     """
     for input_feature in config.get("input_features", []):
         input_feature["saved_weights_in_checkpoint"] = True
+
+
+def processify(func):
```

Comment:
My concern with adding a functionality like this is that it will likely only be used for `monitor`, as in the rest of the codebase we use multiprocessing directly.

Is it possible to refactor `Tracker` so that `monitor` is launched in a separate process without needing processify?